PYTHON2/day20/sday20.py

```diff
@@ -0,0 +1,11 @@
+import re
+str=raw_input()
+str=str.replace('[',' ')
+str=str.replace(']',' ')
+str=str.replace('\\',' ')
+#print str
+s=re.split("!|[|,|?|.|_|'|@|+|]| ",str.strip())
```

Comment:
If you have already replaced `[` and `]` with spaces, why do we need again in regex too?